specification/cosmos-db/resource-manager/readme.md

```diff
@@ -27,7 +27,34 @@ These are the global settings for the Cosmos-DB API.
 ``` yaml
 title: CosmosDBManagementClient
 openapi-type: arm
-tag: package-preview-2024-12-01
+tag: package-preview-2025-05-01
+```
+
+### Tag: package-preview-2025-05-01-preview
+
+These settings apply only when `--tag=package-preview-2025-05-01-preview` is specified on the command line.
+
+```yaml $(tag) == 'package-preview-2025-05-01-preview'
```

Comment:
not same as line 30

Reply:
@tadelesh Resolved.

Reply:
These still don't match.

Line 30 says "build tag `package-preview-2025-05-01` by default" but this _yaml tag_ doesn't exist.

Line 37 defines the _yaml tag_ as `package-preview-2025-05-01-preview`. Note the extra `-preview` suffix. This should be `package-preview-2025-05-01`.